### PR TITLE
git-for-each-repo: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #3953

### CODE REVIEW

File: pages/common/git-for-each-repo.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# git for-each-repo
+
+> Run a Git command on a list of repositories.
+> This command is experimental. **The behavior may change**.
```
Comment (bl-ue) [inline pages/common/git-for-each-repo.md:4]:
```suggestion
> This command is experimental. **The behavior may change**.
> More information: <https://git-scm.com/docs/git-for-each-repo>.
```

Comment (navarroaxel) [reply pages/common/git-for-each-repo.md:4]: OMG, I looked that page and I didn't find it.

File: pages/common/git-for-each-repo.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> More information: <https://git-scm.com/docs/git-for-each-repo>.
+
+- Run maintenance on each of a list of repositories stored in a `maintenance.repo` configuration variable:
```
Comment (bl-ue) [inline pages/common/git-for-each-repo.md:7]:
```suggestion
- Run maintenance on each of a list of repositories stored in the `maintenance.repo` user configuration variable:
```

File: pages/common/git-for-each-repo.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+`git for-each-repo --config={{maintenance.repo}} {{maintenance run}}`
+
+- Run `git pull` on each repository from a specific configuration variable:
+
+`git for-each-repo --config={{configuration_variable}} {{pull}}`
```
Comment (bl-ue) [inline pages/common/git-for-each-repo.md:13]:
```suggestion
- Run `git pull` on each repository listed in a global configuration variable:

`git for-each-repo --config={{global_configuration_variable}} {{pull}}`
```

**Review by bl-ue — COMMENTED** [review]
Left some comments.  Also, if possible, could you please add an example for running an arbitrary command (i.e. not a `git` subcommand)

**Comment by navarroaxel** [comment]
> Left some comments. Also, if possible, could you please add an example for running an arbitrary command (i.e. not a `git` subcommand)

The man page says this is only for git commands.
```
DESCRIPTION
Run a Git command on a list of repositories.
```

But maybe we can add a shell command as a git subcommand alias and that could work. :thinking:

**Comment by bl-ue** [comment]
Oh, I didn't notice that part :). No, it's fine how it is, I think.

**Review by bl-ue — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Pull request merged**